apps/dotcms-ui/.storybook/preview-head.html

```diff
@@ -0,0 +1,2 @@
+<link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet" />
```

Comment:
The pallete show without animation (just how and hide) so it looks weird.

<img width="504" alt="Screen Shot 2021-08-12 at 8 31 58 AM" src="https://user-images.githubusercontent.com/751424/129215414-71b22d4b-8926-4356-a268-d1031aa8a8f1.png">


When I dropped the content type there is `z-index` problem
<img width="1926" alt="Screen Shot 2021-08-12 at 8 35 11 AM" src="https://user-images.githubusercontent.com/751424/129215849-1786be47-87be-426f-a060-9910dcfb0ce5.png">